linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pwquality/accounts_password_pam_pwquality_password_auth/rule.yml

```diff
@@ -29,6 +29,12 @@ references:
 
 ocil_clause: 'pam_pwquality.so is not enabled in password-auth'
 
+ocil: |-
+    To check if pam_pwhistory.so is enabled in password-auth, run the following command:
```

Comment:
also mixing pwquality and pwhistory here

Reply:
Thanks for catching @vojtapolasek . It came wrong from the source and I didn't notice. I am fixing.